.travis.yml

```diff
@@ -45,8 +47,20 @@ matrix:
           compiler: gcc
           env: GCC_VERSION=6
 
+    exclude:
+        - compiler: gcc
```

Comment:
Keep the same ordering as for includes: os, then compiler.

Reply:
👍